HISTORY.md

```diff
@@ -15,6 +15,11 @@
 * Fix the bug that WriteBatchWithIndex's SeekForPrev() doesn't see the entries with the same key.
 * Fix the bug where user comparator was sometimes fed with InternalKey instead of the user key. The bug manifests when during GenerateBottommostFiles.
 
+## 5.17.1 (10/30/2018)
+## Unreleased
+### New Features
+* Add xxhash64 checksum support
```

Comment:
this is the only line that's needed, and it should be moved to the existing unreleased new features section.

Reply:
I should have been clearer in my previous comment, and I have made another PR (#4627)